### ISSUE TITLE
Propose changing name to "Vmoto and Super Soco"

### ISSUE THREAD

**Issue by apbarratt** [drakhart/ha-super-soco-custom#20]
With the name Vmoto seeming to be rolling out more prominently globally (even here in the UK, Super Soco has changed its name to Vmoto UK) I wonder if perhaps we should reflect this name in the integration?

We now even have two versions of the Vmoto app on the go, with it looking like the one simply called "Vmoto" placing itself to supersede the "Vmoto Soco" app and it seems clear that the old "Super Soco" app and ECUs are no longer being developed.

In fact it seems the official Super Soco app doesn't support access outside of the UK at all anymore (due to the region list for selecting phone number area codes having vanished for quite some time now), which may mean this integration may be the only maintained version that provides support (something I hope we continue to do) and not leave folk with no smart option at all as is now the case with older Nissan Leafs.

I'd propose we change the project name to "Vmoto and Super Soco Home Assistant Integration" and the HACS listing and displayed name within Home Assistant to simply "Vmoto and Super Soco".

I would, however, assume that changing the actual repository name and handling any 301 redirects for it might be complicated at best so I'd suggest the actual repository slug name remain as `ha-super-soco-custom` to maintain continuity there.

Happy to create a PR for that if folk are on board?

**Comment by drakhart** [comment]
I agree with the naming update to "Vmoto and Super Soco Home Assistant Integration". 👍 

I'm not worried about 301 redirects in Github since (at least in theory 🤞) they're handled automagically whenever you change a repository name, but tbh I'm a bit concerned about the HACS entry and the icons in the official HA repository, as it might break something there.

Please feel free to submit a PR changing the name in the repo contents (thanks in advance!) and we can look into the consequences of changing the repo name later on.

**Comment by apbarratt** [comment]
Oh good point, I'll have a look at HACS docs to see if there's any guidelines on renaming stuff.

**Comment by drakhart** [comment]
Unfortunately it looks like the Super Soco API has been completely removed from remote servers: methods such as `GET /v1/user/get` or `GET /v1/deviceWarn/list` return a 404 Not Found, and the `POST /v1/login` endpoint simply returns an `ok` string instead of the usual JSON containing a JWT.

I'll remove all the related code from the integration, leaving just the Vmoto API and flows, and take the chance to start the renaming.

**Comment by drakhart** [comment]
Renaming finished, including all repo contents and the repository name.

I opened [this MR for renaming the repo in HACS](https://github.com/hacs/default/pull/9305) as well, but had to close it because [apparently you cannot rename an integration](https://github.com/hacs/integration/issues/5157#issuecomment-4055071221) there.

Thankfully HACS store seems to pick up the redirection without issues: I've been able to re-download the integration and open the repository link via the three-dots-menu without any issues.
